cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We can remove this if we are not using it

Reply:
Removed! gracias caballero